tasks/deploy_start.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Load vars
+  include_vars:
+    file: "{{ playbook_dir }}/vendor/roles/slack-notify/vars/main.yml"
```

Comment:
had to update this to a relative path to fix an issue with us referencing the package/role with different names (`trellis-slack-deploy-notifications` vs `slack-notify`) - is that the name you're using in `galaxy.yml`?
```diff
  include_vars:
-  file: "{{ playbook_dir }}/vendor/roles/slack-notify/vars/main.yml"
+  file: "../vars/main.yml"
```

Reply:
Yeah whoops that's my bad, but I struggled with relative file locations here. It would always pick up the deploy role instead of this role, which kinda makes sense seeing as that's the top level role that's being run. 

**It needs to be the full path**, but you're right I'd changed the name in `galaxy.yml`, so the path therefore would be incorrect.

```yaml
  - name: slack-notify
    src: https://github.com/smithfield-studio/trellis-slack-webhook-notify-during-deploy
    version: 1.0.3
```

I'll commit a change to resolve.